src/seoChecks/index.js

```diff
@@ -0,0 +1,49 @@
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const fs = require('fs');
+const open = require('open');
+const moment = require('moment');
+const { red } = require('../constants/colors');
+const ReportGenerator = require('lighthouse/lighthouse-core/report/report-generator');
+
+const reportHtmlPath = `./ceoReports/ceoReport-${moment().format()}.html`;
+const reportCsvPath = `./ceoReports/ceoReport-${moment().format()}.csv`;
+
+function launchChromeAndRunLighthouse(url, opts, config = null) {
+  return chromeLauncher.launch({ chromeFlags: opts.chromeFlags }).then(chrome => {
+    opts.port = chrome.port;
+    return lighthouse(url, opts, config).then(results => {
+      const res = results.report;
+      // workaround to save the report
+      fs.writeFile(reportHtmlPath, res, err => {
+        if (err) {
+          console.log(err);
+        }
+      });
+      const csv = ReportGenerator.generateReport(results.lhr, 'csv');
+      fs.writeFile(reportCsvPath, csv, err => {
+        if (err) {
+          console.log(err);
+        }
+      });
+      return chrome.kill().then(() => results.lhr);
+    });
+  });
+}
+
+const opts = {
+  output: ['html', 'csv'],
+  chromeFlags: ['--headless'],
+  view: true
+};
+
+module.exports = seoLink => {
+  if (typeof seoLink === 'string') {
```

Comment:
You are creating `seoLink` as a `''` is this going to fail in some case?

Reply:
yes but how you check its a valid url?

Reply:
I think it will always be a string the `-l`
Maybe if you initiate the `seoLink` as undefined you can check if the seoLink exists and throw an error. What i mean is that i don't think this if will ever be false. Beacuse you are initiating seoLink like a string. And if the send something via param it will also be a string